### PR TITLE
Fixed null pointer exceptions from a null JungleBoard in pieces.

### PR DESCRIPTION
**Pull request by leejr0**
Also changed some variable types to match current schema.

### CODE REVIEW

File: server/src/main/java/com/jungleapp/cs414/server/PieceAdapter.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.jungleapp.cs414.server;
```
Comment (vstepanuga) [inline server/src/main/java/com/jungleapp/cs414/server/PieceAdapter.java]: Should not be pushed

File: target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst [hunk 2]
```diff
@@ -1,3 +1,4 @@
+com/jungleapp/cs414/server/PieceAdapter.class
```
Comment (vstepanuga) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst:1]: Should not be pushed

**Review by vstepanuga — CHANGES_REQUESTED** [review]
Some files should not be committed, but otherwise looks good.

File: server/src/main/java/com/jungleapp/cs414/server/PieceAdapter.java (hunk 1 above)
Comment (leejr0) [reply server/src/main/java/com/jungleapp/cs414/server/PieceAdapter.java]: Removed file.

File: target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst (hunk 2 above)
Comment (leejr0) [reply target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst:1]: Removed class.

**Pull request merged**